samples/cellular/nrf_cloud_multi_service/src/cloud_connection.c

```diff
@@ -527,11 +527,6 @@ static int setup_cloud(void)
 		LOG_ERR("Error initializing FOTA: %d", err);
 		return err;
 	}
-	err = coap_fota_begin();
```

Comment:
This will cause `nrf_cloud_fota_poll_process_pending` to be called before `nrf_cloud_init` won't it?

I don't remember why that is a problem, I am pretty sure it has caused us issues in the past

Reply:
this is for CoAP and REST FOTA.
nrf_cloud_init is MQTT.

Reply:
Ah, you're right, my bad